api/api.go

```diff
@@ -152,6 +152,7 @@ type Vehicle interface {
 	Battery
 	BatteryCapacity
 	IconDescriber
+	Id() string
```

Comment:
Das gehört da nicht rein. Die Fahrzeuge sollten nicht wissen, dass sie in einem Slice hängen. Ich hab eine Idee wie wir das umgehen könnten.

Reply:
Ist wieder entfernt.